courses/models.py

```diff
@@ -1119,7 +1119,7 @@ def to_dict(self):
     def grade_percent(self):
         """Returns the grade field value as a number out of 100 (or None if the value is None)"""
         return (
-            Decimal(self.grade * 100, DecimalContext(prec=2, rounding=ROUND_HALF_EVEN))
+            Decimal(self.grade * 100).quantize(exp=Decimal(1), rounding=ROUND_HALF_EVEN)
```

Comment:
@jkachel, do you think we should still stick with `ROUND_HALF_EVEN` (ref: https://docs.python.org/3/library/decimal.html#rounding-modes).  I would think we would want something like `ROUND_HALF_UP`.

Reply:
I was a bit conflicted about this when I added the rounding in.. half even is more "fair" over a long timescale (across all grades recorded) but less so for an individual learner so half up may be the better choice.